dts/arm/nuvoton/npcx7m6fc.dtsi

```diff
@@ -32,14 +32,20 @@
 	};
 };
 
-&spi_fiu0 {
+&qspi_fiu0 {
+	status = "okay";
+
 	int_flash: w25q40@0 {
-		compatible ="jedec,spi-nor";
+		compatible ="nuvoton,npcx-fiu-nor";
 		/*  4194304 bits = 512K Bytes */
 		size = <0x400000>;
```

Comment:
There's a bit of a trend to define flash sizes like this one with `size = <DT_SIZE_K(512 * 8)>`, as in: 512kB in bits. What do you think about adopting that while we are at it? I find it less ambiguous, and you could drop the comment above.

Reply:
It sounds good. I will update it later.

Reply:
Done.